### PR TITLE
add kubeadm ClusterConfiguration  usage

### PR DESCRIPTION
**Pull request by Coderhypo**
fix https://github.com/kubernetes/website/issues/15171

@sftim Hello, I made some improvements, I don’t know if it’s suitable.

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md [hunk 1]
```diff
@@ -21,10 +21,15 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
+3.  Run `kubeadm init` with `--config <YOUR CONFIG YAML>`.
 
 For more details on each field in the configuration you can navigate to our
 [API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2#ClusterConfiguration).
 
+{{< note >}}
+You can get default `ClusterConfiguration` object by `kubeadm config print init-defaults`.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md]:
```suggestion
You can get a starting point ClusterConfiguration by running `kubeadm config print init-defaults` and saving the output to a file of your choice.
```
Any better @Coderhypo ? What do you think?

Comment (Coderhypo) [reply content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md]: Yes, great. thanks.

File: content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md [hunk 2]
```diff
@@ -21,10 +21,15 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
+3.  Run `kubeadm init` with `--config <YOUR CONFIG YAML>`.
 
 For more details on each field in the configuration you can navigate to our
 [API reference pages](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2#ClusterConfiguration).
 
+{{< note >}}
+You can get a starting point ClusterConfiguration by running `kubeadm config print init-defaults` and saving the output to a file of your choice.
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md]: please change to:

> You can generate a `ClusterConfiguration` object with default values by running  `kubeadm config print init-defaults` and saving the output to a file of your choice.

Comment (Coderhypo) [reply content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md]: Done，thanks

**Comment by Coderhypo** [comment]
@neolit123 Hello, I modified the example yaml in the documentation

**Pull request merged**